nipype/info.py

```diff
@@ -141,7 +141,8 @@ def get_nipype_gitversion():
     'configparser',
     'pytest>=%s' % PYTEST_MIN_VERSION,
     'mock',
-    'pydotplus'
+    'pydotplus',
+    'packaging',
```

Comment:
i think, this would also need to be added to `SETUP_REQUIRES`. you can check by trying to install nipype in a clean environment using `python setup.py install`.